source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,95 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``*Collection.FindOne()`` method. 
+
+..
+   method
+
+The ``FindOne()`` method takes a query document that specifies the
+subset of your collection that you would like to retrieve a document from. If you
+pass the ``FindOne()`` method an empty document, MongoDB will consider your
+entire collection when retrieving your document. For more information on documents
+in the MongoDB Go driver, see our
+:doc:`guide on data formats </fundamentals/data-formats>`.
+
+You must pass the ``FindOne()`` method a ``context.Context`` interface type
+value. For more information on the ``context.Context`` interface and the
+``context`` package, see our :doc:`guide on context </fundamentals/context>`.
+
+..
+   options
+
+You can specify options that modify the behavior of your ``FindOne()`` method.
+One example of an option is **sort**, which orders the subset of your collection
```

Comment:
Based on this wording, does not seem clear whether the server or driver is doing the sorting. Here is one suggestion:

```suggestion
One example of an option is **sort**, which applies an order to the matching documents
```

Maybe there should also be examples of sort documents here? Or if possible, link to other documentation describing what are valid sort documents.

Reply:
Thanks for note. I have changed the wording of this section to be more explicit that your MongoDB instance is performing the sort operation. I have added links to the sort and projections guide at the end of the paragraph. Fix: https://github.com/mongodb/docs-golang/pull/2/commits/0f91e0182facc6e7ed14d46bd00c757cc40768f6